Misc/NEWS.d/next/Documentation/2019-10-01-10-53-46.bpo-22377.5ni-iC.rst

```diff
@@ -0,0 +1 @@
+Improves ``%Z`` in datetime documentation.
```

Comment:
```suggestion
Improves documentation of the values that :meth:`datetime.datetime.strptime` accepts for ``%Z``.
Patch by Karl Dubost.
```

Added a bit more detail to the changelog here, and also added your name. If you don't want to be credited, feel free to omit the "Patch by.." portion.

Reply:
thx